From 3842cd28126855628254e0e2701d988e38fc34d8 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sun, 26 Oct 2008 14:46:25 +0200
Subject: [PATCH] charset=iso-8859-1 to utf-8

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 micro_httpd.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/micro_httpd.c b/micro_httpd.c
index c87034a..2ebbd48 100644
--- a/micro_httpd.c
+++ b/micro_httpd.c
@@ -197,9 +197,9 @@ get_mime_type( char* name )
 
     dot = strrchr( name, '.' );
     if ( dot == (char*) 0 )
-	return "text/plain; charset=iso-8859-1";
+	return "text/plain; utf-8";
     if ( strcmp( dot, ".html" ) == 0 || strcmp( dot, ".htm" ) == 0 )
-	return "text/html; charset=iso-8859-1";
+	return "text/html; utf-8";
     if ( strcmp( dot, ".jpg" ) == 0 || strcmp( dot, ".jpeg" ) == 0 )
 	return "image/jpeg";
     if ( strcmp( dot, ".gif" ) == 0 )
@@ -228,7 +228,7 @@ get_mime_type( char* name )
 	return "application/ogg";
     if ( strcmp( dot, ".pac" ) == 0 )
 	return "application/x-ns-proxy-autoconfig";
-    return "text/plain; charset=iso-8859-1";
+    return "text/plain; utf-8";
     }
 
 
-- 
1.6.5

