From c7edcfdbf962f2135b09b8ce8388233cd9500d81 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sun, 26 Oct 2008 14:46:41 +0200
Subject: [PATCH] added shockwave flash mime type support for .swf files


Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 micro_httpd.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/micro_httpd.c b/micro_httpd.c
index c87034a..66417d5 100644
--- a/micro_httpd.c
+++ b/micro_httpd.c
@@ -214,6 +214,8 @@ get_mime_type( char* name )
 	return "audio/wav";
     if ( strcmp( dot, ".avi" ) == 0 )
 	return "video/x-msvideo";
+    if ( strcmp( dot, ".swf" ) == 0 )
+	return "application/x-shockwave-flash";
     if ( strcmp( dot, ".mov" ) == 0 || strcmp( dot, ".qt" ) == 0 )
 	return "video/quicktime";
     if ( strcmp( dot, ".mpeg" ) == 0 || strcmp( dot, ".mpe" ) == 0 )
-- 
1.6.5

