From 917abd26c62787e87b959cae77dd3372d0eddd1d Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sun, 26 Oct 2008 14:46:49 +0200
Subject: [PATCH] Remove strip


Signed-off-by: Jari Aalto <jari.aalto@cante.net>
Signed-off-by: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index 090cadc..2faa776 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ MANDIR =	/usr/local/man/man8
 CC =		cc
 CFLAGS =	-O -ansi -pedantic -U__STRICT_ANSI__ -Wall -Wpointer-arith -Wshadow -Wcast-qual -Wcast-align -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -Wredundant-decls -Wno-long-long
 #SYSVLIBS =	-lnsl -lsocket
-LDFLAGS =	-s $(SYSVLIBS)
+LDFLAGS =	$(SYSVLIBS)
 
 all:		micro_httpd
 
-- 
1.6.5

